app/controllers/billboards_controller.rb

```diff
@@ -37,6 +37,6 @@ def placement_area
   def user_tags
     return unless feed_targeted_tag_placement?(placement_area)
 
-    current_user.cached_followed_tag_names
+    current_user&.cached_followed_tag_names
```

Comment:
This is unrelated but I wanted to test it out for a test that was failing.

Reply:
Hi @Ridhwana Any updated comment on this?

Reply:
The test is no longer failing on this branch (although I'm not able to reliably re-produce the flakiness), hence I'd like to merge it into main.